### DISCUSSION TITLE
Watch commit

### DISCUSSION THREAD

**Discussion by jagama** [community/community#4950] [Mobile]
Possibility to watch commit on specific branch of a repo and get notified when a collaborator push any changes ?

**Comment by eliperkins** [comment]
Thanks for the feedback @jagama! I'd love to hear a little bit more about your use-case. For my understanding, could you explain a little more to me about how you might use this feature? What are some repos whose branches you watch for commits for?

**Reply by jagama** [reply]
Hi, 
thanks for the quick response. 

I have a business account, and in my repo two or three collaborators are pushing changes into different branches. 

I'm the CEO and Tech Leader of an IT Company and every time my collab push changes I have to check all the commits and the work done - or just to see what they have done. 

I'm always moving for various reasons and the ability to check the commits on the go would be a great feature. 

Se my suggestion is simple: get notified when my collaborator pushed some changes to GitHub (any or specific branch) and check it rapidly on the fly. 

Thanks for the attention and greetings from Italy. 

J

**Reply by KlausEverWalkingDev** [reply]
I think this is great. We already can watch for issues, pull requests and releases. As not always people work through pull request and releases, in some projects, specially the small ones, I tend to think they are stale, when they are not. Frequently they just are only pushing their work through commits. And until now there is not notification nor option to watch that specific case.
